kore/test/Test/Kore/Step/Axiom/Matcher.hs

```diff
@@ -71,6 +71,9 @@ import qualified Test.Kore.Step.MockSymbols as Mock
 import Test.Kore.Step.Simplification
 import Test.Tasty.HUnit.Ext
 
+import Debug.Trace
+import Kore.Unparser
+
```

Comment:
As Tom said below, I think you forgot to remove these.

Reply:
Oops, fixed.